MOF-808/lmp_umbrella_sample

```diff
@@ -0,0 +1,266 @@
+pair_coeff      3      4         lj/cut      0.086000      3.400000
+pair_coeff      3      5         lj/cut      0.035917      3.000000
+pair_coeff      3      3         lj/cut      0.086000      3.400000
+pair_coeff      3      9         lj/cut      0.134387      3.180000
+pair_coeff      9     11         lj/cut      0.210000      2.960000
+pair_coeff      9     10         lj/cut      0.210000      2.960000
+pair_coeff      9      9         lj/cut      0.210000      2.960000
+
+
+## TIP4P H2O ##
+#pair_coeff      12 12 lj/cut/tip4p/long 0.1852 3.1589
+#pair_coeff      12 13 lj/cut/tip4p/long 0.0 0.0
+#pair_coeff      13 13 lj/cut/tip4p/long 0.0 0.0
+
+## Mixed ##
+pair_coeff      1     12         lj/cut      0.136360      3.023465
+pair_coeff      1     13         lj/cut      0.000005      2.733270
+pair_coeff      6     12         lj/cut      0.000000      0.000000
+pair_coeff      6     13         lj/cut      0.000000      0.000000
+pair_coeff      7     12         lj/cut      0.063578      2.842665
+pair_coeff      7     13         lj/cut      0.000002      2.552470
+pair_coeff     11     12         lj/cut      0.237888      3.111965
+pair_coeff     11     13         lj/cut      0.000009      2.821770
+pair_coeff     10     12         lj/cut      0.237888      3.111965
+pair_coeff     10     13         lj/cut      0.000009      2.821770
+pair_coeff      2     12         lj/cut      0.152234      3.331965
+pair_coeff      2     13         lj/cut      0.000006      3.041770
+pair_coeff      8     12         lj/cut      0.237888      3.111965
+pair_coeff      8     13         lj/cut      0.000009      2.821770
+pair_coeff      4     12         lj/cut      0.152234      3.331965
+pair_coeff      4     13         lj/cut      0.000006      3.041770
+pair_coeff      5     12         lj/cut      0.063578      2.931965
+pair_coeff      5     13         lj/cut      0.000002      2.641770
+pair_coeff      3     12         lj/cut      0.152234      3.331965
+pair_coeff      3     13         lj/cut      0.000006      3.041770
+pair_coeff      9     12         lj/cut      0.237888      3.111965
+pair_coeff      9     13         lj/cut      0.000009      2.821770
+
+#MBX-water
+pair_coeff      * * mbx  0.0 0.0
+pair_coeff      12      12 mbx  0.0 0.0
+pair_coeff      12      13 mbx  0.0 0.0
+pair_coeff      13      13 mbx  0.0 0.0
+
+#coul exclude
+pair_coeff      1*11 1*11 coul/exclude
+
+## Non-bonded k-space ##
+#kspace_style    ewald 0.000001
+#kspace_style    pppm/tip4p 1.0e-5
+#special_bonds   amber
+
+
+neighbor        2.0 bin
+neigh_modify	every 1 delay 1 check yes
+
+timestep        ${dt}
+compute         mbx all pair mbx
+variable        e1    equal c_mbx[1]
+variable        e2    equal c_mbx[2]
+variable        e3    equal c_mbx[3]
+variable        ebuck equal c_mbx[4]
+variable        edisp equal c_mbx[5]
+variable        eele  equal c_mbx[6]
+variable        etot  equal c_mbx[7]
```

Comment:
This is the old mbx variable style. Unless you are using a version of MBX that is over a year old, this should be updated.